api/client/admin/admin.go

```diff
@@ -1,21 +1,21 @@
 package admin
 
 import (
-	proto "github.com/textileio/powergate/proto/admin/v1"
+	adminPb "github.com/textileio/powergate/api/gen/powergate/admin/v1"
 )
 
 // Admin provides access to Powergate admin APIs.
 type Admin struct {
 	StorageJobs *StorageJobs
-	Profiles    *Profiles
+	Users       *Users
```

Comment:
New "user" wording in the client. Feels much easier and makes sense.

Reply:
I agree that user feels weird, but I definitely prefer it over storage profile.